addons-l10n/en/hovercards.json

```diff
@@ -0,0 +1,28 @@
+{
+  "hovercards/post-count": "{count} posts",
+  "hovercards/loading": "Loading...",
+  "hovercards/error-request-failed": "Request failed. {error}",
+  "hovercards/error-scratchdb": "Error encountered on ScratchDB. {error}",
```

Comment:
This error should be more clear, e.g. "ScratchDB server is having a problem: {error}"